test/parallel_api/algorithm/alg.sorting/alg.min.max/minmax_element_coal.pass.cpp

```diff
@@ -0,0 +1,23 @@
+// -*- C++ -*-
+//===-- minmax_element_coal.pass.cpp --------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#define _ONEDPL_DETECT_SPIRV_COMPILATION 1
```

Comment:
To test the coalesced implementation, should `_ONEDPL_DETECT_SPIRV_COMPILATION ` be set to 0? This would be the case where the coalesced reduce implementation should be invoked I believe.

Reply:
Yes, you are right. This error is also present in `minmax_element_pass.cpp`. I've updated the PR with a fix.